.github/workflows/os-osd-deployment-scheduled.yml

```diff
@@ -0,0 +1,34 @@
+# This workflow is a scheduled workflow and used for deploying the preview version of OpenSearch
+# and OpenSearch Dashboards to AWS EKS (Elastic Kubernetes Service) cluster.
+name: Scheduled Deploy OpenSearch and OpenSearch Dashboards
+
+on:
+  # Adds workflow_dispatch for manually running a workflow
+  workflow_dispatch:
+  schedule:
+    # Run every night at 21:30 PDT.
```

Comment:
Is it PDT?

Reply:
Good catch, the default timezone is UTC, i changed the daily time to map the UTC

Reply:
> Open discussions(not a blocker): Iooks like we need to have a new folder and a new set of configs for a new cluster, I wonder how we plan to manage different clusters in the future and what are their use cases and how to manage it better - maybe having an object store for the yaml files, then the workflow just access the object store and create/update cluster

If we are adding object storage, we need to enable public access for the object store, the main reason list here is that we need to demonstrate public how to use the helm chart with proper configuration to deploy.

Reply:
ok, yeah, I think for demonstration, it is okay, and we can think about long term later